AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.m

```diff
@@ -9,6 +9,16 @@
 
 @interface MSAuthTokenContext ()
 
+/**
+ * Authorization token cached value.
+ */
+@property(nonatomic) NSString *authToken;
+
+/**
+ * The last value of user account id.
+ */
+@property(nonatomic) NSString *lastHomeAccountId;
```

Comment:
`lastHomeAccountId` naming is a little inconsistent with getter/setter of the auth token. Maybe we just use `homeAccountId` for consistency purpose?